modules/nf-core/t1k/run/meta.yml

```diff
@@ -0,0 +1,177 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "t1k_run"
+description: write your description here
+keywords:
+  - polymorphic
+  - genes
+  - HLA
+  - Kir
+tools:
+  - "t1k":
+      description: "T1K is a versatile methods to genotype highly polymorphic genes
+        (e.g. KIR, HLA) with RNA-seq, WGS or WES data."
+      homepage: "https://github.com/mourisl/T1K/blob/v1.0.8/README.md"
+      documentation: "https://github.com/mourisl/T1K/blob/v1.0.8/README.md"
+      tool_dev_url: "https://github.com/mourisl/T1K"
+      doi: "10.1101/gr.277585.122"
+      licence: ["MIT"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample id and single_end label for fastq input files.
+          e.g. `[ id:'sample1', single_end:false ]`
+    - input:
+        type: file
+        description: BAM or FASTQ file(s).
```

Comment:
Maybe mention that it can be either a single bam, two paired-end fastq, an interleaved fastq or a single-end fastq, and anything else will error?